packages/ai-chat-ui/src/browser/chat-response-renderer/delegation-response-renderer.tsx

```diff
@@ -115,12 +115,12 @@ class DelegatedChat extends React.Component<DelegatedChatProps, DelegatedChatSta
         let statusIcon = '';
         let statusText = '';
         if (hasNode) {
-            if (isComplete) {
+            if (isCanceled) {
+                statusIcon = 'codicon-close';
+                statusText = 'canceled';
```

Comment:
The status texts are not internationalized with `nls`, also the other labels and messages below should be aligned.